tests/phpunit/remote-config/class-wc-stripe-remote-config-scheduler-test.php

```
@@ -0,0 +1,132 @@
+			'flags'        => [ 'optimized_checkout' => [ 'value' => $value ] ],
+			'generated_at' => '2026-05-09T12:00:00Z',
+		];
+	}
+
+	public function test_init_hooks_registers_action_callback(): void {
+		$scheduler = new WC_Stripe_Remote_Config_Scheduler();
+		$scheduler->init_hooks();
+
+		$this->assertNotFalse( has_action( WC_Stripe_Remote_Config_Scheduler::SYNC_ACTION, [ $scheduler, 'run' ] ) );
+		$this->assertNotFalse( has_action( 'woocommerce_stripe_updated', [ $scheduler, 'on_plugin_upgrade' ] ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_init_hooks_registers_action_callback is not named in camelCase. (undefined)

(CamelCaseMethodName)